scripts/generate-conda-recipe.py

```diff
@@ -0,0 +1,73 @@
+import yaml
+
+class RecipeDumper(yaml.SafeDumper):
+    """Adds a line break between top level objects and ignore aliases"""
+
+    def write_line_break(self, data=None):
+        super().write_line_break(data)
+        if len(self.indents) == 1:
+            super().write_line_break()
+
+    def ignore_aliases(self, data):
+        return True
+
+    def increase_indent(self, flow=False, *args, **kwargs):
+        return super().increase_indent(flow=flow, indentless=False)
+
+def delete_lines(file: str, line: int):
+    """Delete a line from the file head"""
+    with open(file, 'r') as fr:
+        read_lines = fr.readlines()
+
+    with open(file, 'w') as fw:
+        write_lines = ''.join(read_lines[line:])
+        fw.write(write_lines)
+
+
+def get_lines(file: str, start: int, end: int):
+    """Get lines from the file"""
+
+    with open(file, 'r') as fr:
+        read_lines = fr.readlines()
+    return read_lines[ start:end ]
+
+
+def generate_meta_yaml(source_path: str, target_path: str):
+    """Generate conda meta yaml"""
+
+    save_lines = get_lines(source_path, 0, 2)
+    delete_lines(source_path, 2)
+
+    with open(source_path) as f:
+        meta_dict = yaml.safe_load(f)
+    
+        meta_dict['package'] = {'name': '<{ name|lower }>', 'version': '<{ version }>'}
+
+        # build noarch package
+        meta_dict['build']['noarch'] = 'python'
+        if(meta_dict['requirements'] and meta_dict['requirements']['host']):
+            meta_dict['requirements']['host'] = ['docker-py' if item == 'docker' else item for item in meta_dict['requirements']['host']]
+        if(meta_dict['requirements'] and meta_dict['requirements']['run']):
+            meta_dict['requirements']['run'] = ['docker-py' if item == 'docker' else item for item in meta_dict['requirements']['run']]
```

Comment:
I'm no python expert, but I think this is unclear.

Maybe try the GitHub Copilet trick from Florian. 
https://jinaai.slack.com/archives/C01L8SPGAFP/p1666775786003009

Reply:
You are right, I create a function looks like below: to replace it
```
def replace_package_name(requirements: list, old: str, new: str):
    """replace package name"""

    replace_requirements = []
    for package_name in requirements:
        if package_name == old:
            replace_requirements.append(new)
        else:
            replace_requirements.append(package_name)
    return replace_requirements
```